lib/components/stream_preview.dart

```diff
@@ -20,17 +20,24 @@ class StreamPreview extends StatefulWidget {
 
 class _StreamPreviewState extends State<StreamPreview> {
   WebViewController? _controller;
+  String? url;
 
   var _isOverlayActive = false;
   Timer? _overlayTimer;
 
   @override
   Widget build(BuildContext context) {
+    final newUrl =
+        'https://player.twitch.tv/?channel=${widget.channelDisplayName}&controls=false&parent=chat.rtirl.com&muted=false';
+    if (url != newUrl && _controller != null) {
+      _controller!.loadUrl(newUrl);
```

Comment:
`build()` shouldn't have side effects, can this go in https://api.flutter.dev/flutter/widgets/State/didUpdateWidget.html instead?

Reply:
Nice, thanks!